xkbregistry.map

```diff
@@ -0,0 +1,60 @@
+/* versions are kept in sync with libxkbcommon.so */
```

Comment:
ack, needs special treatment though since a lot of the functions are macro-generated and don't show up in grep. Need to think about this.

Reply:
TBH I'm not a huge fan of the function-generating macros, because they break grep and jump-to-definition and such. Sure there's a lot of duplication but I'm not sure it really matters. (But I totally defer to your discretion on that).

Reply:
We can expand the macros to normal functions once we're happy with the code. For now it's a lot easier to change one macro than to update 10 `ref/unref` calls :)